### PR TITLE
Feature/7247 job variations update modal support field update

### PR DESCRIPTION
**Pull request by s3341458**

### CODE REVIEW

File: typescript/src/entities/email_address.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export class EmailAddress extends Entity {
   protected static singularName: string = "emailAddress";
   protected static pluralName: string = "emailAddresses";
 
-  @EmailAddress.property({type: EmailAddress})
+  @EmailAddress.property({type: Date})
```
Comment (Amtron) [inline typescript/src/entities/email_address.ts]: how come?

File: typescript/src/entities/job.ts [hunk 2]
```diff
@@ -193,23 +195,35 @@ export class Job extends Entity {
...
+    const resource = '/specialised-order-estimate/';
+    const data = this.toFormData({excludeOld: false});
+    const fetchOptions: RequestOptions = {method: 'POST', body: data};
+    fetchOptions.query = [];
+    fetchOptions.query.push(['skip_rights', 'y']);
+
+    return this.merchi.authenticatedFetch(resource, fetchOptions).
+      then((data: any) => { this.fromJson(data, {makeDirty: true});
+        return this;});
+  }
```
Comment (Amtron) [inline typescript/src/entities/job.ts]: I like this.

**Comment by Amtron** [comment]
@s3341458 this has errors.

File: typescript/src/entities/email_address.ts (hunk 1 above)
Comment (s3341458) [reply typescript/src/entities/email_address.ts]: previously was Wrong, Date was the right one which maps to archived

**Pull request merged**